runtime/src/iree/hal/drivers/hip/hip_allocator.c

```diff
@@ -15,11 +15,28 @@
 #include "iree/hal/drivers/hip/hip_buffer.h"
 #include "iree/hal/drivers/hip/per_device_information.h"
 #include "iree/hal/drivers/hip/status_util.h"
+#include "iree/hal/drivers/hip/util/queue.h"
+#include "iree/hal/drivers/hip/util/tree.h"
 
 #if IREE_TRACING_FEATURES & IREE_TRACING_FEATURE_ALLOCATION_TRACKING
 static const char* IREE_HAL_HIP_ALLOCATOR_ID = "HIP unpooled";
 #endif  // IREE_TRACING_FEATURE_ALLOCATION_TRACKING
 
+#define iree_hal_hip_async_allocation_default_queue_size 8
```

Comment:
```suggestion
#define IREE_HAL_HIP_ASYNC_ALLOCATION_DEFAULT_QUEUE_SIZE 8
```
style: all caps for macro literals
https://google.github.io/styleguide/cppguide.html#Macro_Names